scenes/test.scn

```diff
@@ -0,0 +1,40 @@
+{
+  "objects": [
+    {
+      "type": "application/light",
+      "content": {
+        "lightType": "ambient",
+        "args": [[255, 255, 255], 2]
+      }
+    },
+    {
+      "type": "application/light",
+      "content": {
+        "lightType": "directional",
+        "args": [[255, 255, 255], 2],
+        "position": [1, 2, 3]
+      }
+    },
+
+    { "position": [1, 1, -1], "quaternion": [0, 0.7071067811865475, 0, 0.7071067811865476], "start_url": "http://localhost/silsword/" },
+
+    {
+      "position": [ 0, 3, -3 ],
+      "quaternion": [ 0, 0.7071067811865475, 0, 0.7071067811865475 ],
+      "type": "application/npc",
+      "content": {
+        "name": "Ann",
+        "avatarUrl": "/avatars/ann.vrm",
+        "voice": "Maud Pie",
+        "bio": "13/F Lisk Witch (black mage). She is 100% quirky and is always creating crazy new potions. She has a dark streak and is known to torment her teachers when they give her bad grades.",
+        "wear": [
+          {
+            "start_url": "http://localhost/silsword/"
```

Comment:
Add a new `.scn` such as `combat.scn`, put into `scenes.json`.
Using formal links instead of `localhost`.